filesystems/blob/src/main/scala/cromwell/filesystems/blob/BlobPathBuilderFactory.scala

```diff
@@ -23,11 +23,11 @@ final case class BlobPathBuilderFactory(globalConfig: Config, instanceConfig: Co
   val sasToken: String = instanceConfig.as[String]("sas-token")
   val container: String = instanceConfig.as[String]("store")
   val endpoint: String = instanceConfig.as[String]("endpoint")
-  val workspaceId: String = instanceConfig.as[String]("workspace-id")
-  val workspaceManagerURL: String = singletonConfig.config.as[String]("workspace-manager-url")
+  val workspaceId: Option[String] = instanceConfig.as[Option[String]]("workspace-id")
+  val workspaceManagerURL: Option[String] = singletonConfig.config.as[Option[String]]("workspace-manager-url")
 
   val blobTokenGenerator: BlobTokenGenerator = BlobTokenGenerator.createBlobTokenGenerator(
-    container, endpoint, Option(workspaceId), Option(workspaceManagerURL))
+    container: String, endpoint: String, workspaceId: Option[String], workspaceManagerURL: Option[String])
```

Comment:
I don't think we would normally put type annotations here, in fact I didn't even know that was valid code! TIL.